webui.lua

```diff
@@ -593,7 +615,7 @@ local servers = init_servers()
 if passwd ~= 1 then
   if next(servers) == nil then
     error_msg = "Error: Couldn't spawn server on port " .. options.port
-    message = function() mp.msg.error(error_msg); mp.osd_message(MSG_PREFIX .. error_msg, 5) end
+    message = function() mp.msg.error(error_msg); mp.osd_message(MSG_PREFIX .. error_msg, 2) end
```

Comment:
See my other comment about the startup message.

Reply:
sorry, didn't mean to include these local hacks in this PR